src/services/recovery/recovery-state.ts

```diff
@@ -63,6 +56,28 @@ export const _getSafeCreationReceipt = memoize(
   ({ transactionService, safeAddress }) => transactionService + safeAddress,
 )
 
+const queryAddedTransactions = async (
```

Comment:
Can we add a unit test for this?

Reply:
I added more tests for it's outside function effectively also testing this logic.